tests/run/pycontextvar.pyx

```diff
@@ -43,3 +52,5 @@ def test_get_value_no_default(var, default=NOTHING):
     'default'
     """
     return get_value_no_default(var, default) if default is not NOTHING else get_value_no_default(var)
+
+__test__ = {}
```

Comment:
Do we still automatically generate the `__test__` dict? Is this still necessary?

Reply:
I think we do still generate the `__test__` dict.

According to the doctest docs:
> CPython implementation detail: Prior to version 3.4, extension modules written in C were not fully searched by doctest.

I think that means it's still necessary for Python 2.7